modules/admin/proc-creating-secrets-for-postgresql.adoc

```diff
@@ -0,0 +1,54 @@
+[id="proc-creating-secrets-for-postgresql_{context}"]
+= Creating PostgreSQL secrets
```

Comment:
I would suggest to not to separate secrets creation from configuring using Chart/Helm.
Secrets are part of configuration, w/o it configuration procedure looks quite inconsistent and confusing IMHO.